image/v1/tests/ImageStreamImportMode.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Image"
+crdName: images.config.openshift.io
+tests:
+  onCreate:
+   - name: Should be able to create a minimal Image config
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Image
+        spec: {}
+      expected: |
+        apiVersion: config.openshift.io/v1
+        kind: Image
+        spec: {}
+        status:
+          internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
+   - name: Should be able to configure imageStreamImportMode to Legacy
```

Comment:
What about testing for an invalid value?

Reply:
added.